docs/platform-concepts/agi-token.md

```diff
@@ -18,28 +18,28 @@ micro_nav: true
 
-## Testnet AGI vs Mainnet AGI
+## Testnet AGIX vs Mainnet AGIX
 For those new to Ethereum, it may be a bit confusing what exactly the difference between Testnet and Mainnet tokens are. In principle, you can distinguish them as follows:
 * Testnet tokens are used to test software applications with, try out demos, or in our case the Beta.
 * Mainnet tokens are used for officially deployed software.
 * Testnet tokens never have a monetary value.
 * Mainnet tokens could have a monetary value.
 * Both tokens can be used to add extra utility to the software.
 
-Since we are currently in the Beta stage, we only use Kovan and Ropsten Testnet AGI. You can find our Mainnet AGI Token (to be used in a later stage) [here](https://etherscan.io/address/0x8eb24319393716668d768dcec29356ae9cffe285) with the address `0x8eb24319393716668d768dcec29356ae9cffe285`.
+Since we are currently in the Beta stage, we only use Kovan and Ropsten Testnet AGIX. You can find our Mainnet AGIX Token (to be used in a later stage) [here](https://etherscan.io/address/0x8eb24319393716668d768dcec29356ae9cffe285) with the address `0x8eb24319393716668d768dcec29356ae9cffe285`.
 
 <div class="callout callout--warning">
-    <p><strong>NOTE</strong> While we are aware that the AGI token is currently being traded on some exchanges, we do not encourage or facilitate this exchange trading in any manner. Speculative secondary trading is against the spirit of the AGI token and SingularityNET project. We strongly discourage speculative secondary trading and officially ask AGI token holders to act accordingly.</p>
+    <p><strong>NOTE</strong> While we are aware that the AGIX token is currently being traded on some exchanges, we do not encourage or facilitate this exchange trading in any manner. Speculative secondary trading is against the spirit of the AGIX token and SingularityNET project. We strongly discourage speculative secondary trading and officially ask AGIX token holders to act accordingly.</p>
 </div>
 
 
-## AGI Faucet
-We have an automated faucet for distributing Kovan and Ropsten Testnet AGI Tokens [here]({{ site.data.faucets.agi }}). You will need to login using your GitHub account and input the Ethereum address where you want to receive the AGI token. Make sure this address is a Kovan Ethereum Address. You can request 1 AGI token every 24 hours.
+## AGIX Faucet
+We have an automated faucet for distributing Kovan and Ropsten Testnet AGIX Tokens [here]({{ site.data.faucets.agix }}). You will need to login using your GitHub account and input the Ethereum address where you want to receive the AGIX token. Make sure this address is a Kovan Ethereum Address. You can request 1 AGIX token every 24 hours.
 
-In order to add the Kovan Testnet AGI you may need the following details:
-* Symbol: AGI
+In order to add the Kovan Testnet AGIX you may need the following details:
+* Symbol: AGIX
 * Decimals: 8
```

Comment:
Change the address

Reply:
@anandrgitnirman  Please have a look here too...